docs/source/cpp_api/core.rst

```diff
@@ -20,4 +20,5 @@ expose in public APIs.
    core_nvtx.rst
    core_interruptible.rst
    core_operators.rst
-   core_math.rst
\ No newline at end of file
+   core_math.rst
+   core_bitset.rst
```

Comment:
As we continue to build out these filtering primitives, I think we need to consider consolidating them under a single page in the documentation, kind of like what we have done for the math primitives. 

I don't think we need to do that right now, though. But something to think about- maybe something like "fast lookup primitives" or something.